### DISCUSSION TITLE
B48363E7-F82F-6714-123D-F8EF0B6B5D2C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#123] [General]
# Fomalhaut

Fomalhaut is a triple star system consisting of an A-type and K-type main-sequence stars and a red dwarf about 25 light-years away in the constellation Pisces Austrinus. From 2008 to 2020, a [planet was suspected to be forming in the protoplanetary disk](https://celestia.mobi/addon?item=A514784B-ED88-2A10-BD2F-DE786C66C5F4) surrounding the primary star of the system, until it was later determined it is just an expanding dense cloud of debris within the disk itself

https://celestia.mobi/addon?item=B48363E7-F82F-6714-123D-F8EF0B6B5D2C

**Comment by exocritic5457** [comment]
Really makes me wonder why there's no Eps Eri disk addons as of now. ...like this just looks awesome.

**Reply by kmjordan122313-lgtm** [reply]
i really agree.
